CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Package-specific changes not released in any SDK will be added here just before
 
 ### 🎉 New features
 
+- Add isAvailableAsync to expo-task-manager ([#10657](https://github.com/expo/expo/pull/10657) by [@PranshuChittora](https://github.com/pranshuchittora))
+
```

Comment:
```suggestion
```
If there's a package-level CHANGELOG, the change description goes there. This one is filled during the release of the Expo Client 😉